ui/src/views/repositories/components/repositories-table/repositories-table-columns/repository-name.tsx

```diff
@@ -22,48 +22,55 @@ export const RepositoryName: React.FC<RepositoryNameProps> = (props) => {
       case 'gitlab':
       case 'bitbucket':
       case 'other':
-        return <RepositoryIcon className="t-icon text-semantic-mutedIcon w-4" />
+        return <RepositoryIcon className='t-icon text-semantic-mutedIcon w-4' />
       default:
         return (
           <a target='_blank' href={GITHUB_URL + props.name} rel='noopener noreferrer'>
-            <GithubIcon className="t-icon text-semantic-mutedIcon w-4" />
+            <GithubIcon className='t-icon text-semantic-mutedIcon w-4' />
           </a>
         )
     }
   }
 
   return (
-    <div className="flex items-center gap-4 my-3 ml-6">
+    <div className='flex items-center gap-4 my-3'>
       <RepoImage
         repoType={props.type}
         orgName={props.name.split('/')[0]}
-        size="10"
+        size='10'
       />
       <div>
         <Link href={`/repos/${props.id}`}>
-          <h4 data-testid={TEST_IDS.repoNameTable} className="font-medium mb-0.5 text-semantic-text cursor-pointer hover_text-blue-600">
+          <h4 data-testid={TEST_IDS.repoNameTable} className='font-medium mb-0.5 text-semantic-text cursor-pointer hover_text-blue-600'>
             {props.name}
           </h4>
         </Link>
-        <div className="flex items-center">
-          <span className="pr-2 text-sm text-semantic-mutedText">
+        <div className='flex items-center'>
+          <span className='pr-2 text-sm text-semantic-mutedText'>
             <Tooltip
               content={`Added ${format(props.createdAt, 'PPp')}`}
               placement='bottom'
             >
               {getRelativeTime(props.createdAt)}
             </Tooltip>
           </span>
-          <div className="border-l border-semantic-border px-2">
-            {repoTypeIcon()}
+          <div className='border-l border-semantic-border px-2'>
+            <Tooltip
+              content={`${props.type} repository`}
```

Comment:
This appears in the UI as `github repository`:

<img width="413" alt="image" src="https://user-images.githubusercontent.com/57259/195355302-5b731fbd-a873-4b93-a269-3f4620a7641c.png">

Can we tweak this to handle the capitalization? `GitHub repository`

Reply:
Done [eb4649b](https://github.com/mergestat/fuse/pull/391/commits/eb4649bf27c7b3b7f4b96df45b169e9f732f4726)